src/PrestaShopBundle/Resources/views/Admin/Layout/legacy_layout.html.twig

```diff
@@ -96,9 +104,7 @@
              title="{{ maintenanceTitle|e('html') }}"
              href="{{ url('admin_maintenance') }}"
           >
-            <i class="material-icons"
-               style="{% if ps.frontOfficeAccessibleForAdmins|default(false) %}color: #72c279;{% endif %}"
```

Comment:
You need to keep it.

Reply:
I have highlighted the maintenance link in the header in CSS after a talk with designers ✅